content/support.rst

```diff
@@ -0,0 +1,193 @@
+Support
+#######
+
+:menu_order: 009
+
+.. contents::
+
+
+Introduction
+============
+
+This page describes the guidelines for the support activities together with
+best practices that were extracted based on our past experience with support
+related activities.
+
+For access to any of the Support platforms, please contact the Support team.
+
+For the full details on the Support offering, please go to the website's
+`Support options <https://www.sftpplus.com/support/options.html>`_ page.
+
+
+Helpdesk
+========
+
+The main point of contact for Support and liaising with customers
+is via the Google Group email.
+Each message to the Support email automatically creates a helpdesk ticket.
+
+
+Template: Obtaining the installation detail
+-------------------------------------------
+
+The following is a template to obtain the installation detail.
+Please modify the template below to suit.
+
+.. sourcecode:: rst
+
+	Can you please provide us with the following details about the system
+	used for running the product:
+
+	* Product version?
+
+	* Operating system name and version?
+
+	* File transfer protocol(s) used?
+
+	* Please send through a time-stamped version of the server-side log, and
+	   if relevant, the corresponding client-side log.
+
+	* Other information (if known).
+
+	* Please advise brief details of any: integration, software, resilience / redundancy /
+	high availability architecture, security or other technologies that interact with the
+	product.
+	e.g. Firewalls, Proxy, LDAP, Active Directory, Single Sign On, Management software etc.
+
+	Before releasing a new version, we run an automated test suite to validate new features
+	and check for regressions.
+	
+	By knowing details about your installations, we can configure the test suite to run
+	against a similar setup.
+
+	This information will not be shared with any other third party and will be used
+	solely for improving further support for you and development of the product.
+
+
+Using the installation details
+------------------------------
+
+Once the installation details are provided, recreate the issue in a similar
+environment.
+
+This should be created via a dedicated VM for testing / support activities.
+
+
+Chat
+====
+
+`Olark <https://www.olark.com>`_ is currently used for chat with our customers
+or possible customers.
+
+Each chat transcript is automatically forwarded to the Support email list.
+
+A chat can be transferred and other actions can be taken with the chat via
+Olark `commands <https://www.olark.com/help/commands>`_.
+
+
+Licenses and support contract
+=============================
+
+See the `Support options page <https://www.sftpplus.com/support/options.html>`_
+which describes the options available.
+
+If a support case goes beyond the standard option (such as a new
+product feature), the Sales team will determine the quote.
+
+See the `Life cycle page <https://www.sftpplus.com/product/life-cycle.html>`_ 
+for details about the product life cycle.
```

Comment:
Again, the guide should talk about how we handle support, describe what we do and why we do it.

Creating links to the sftpplus.com will not help as the sftpplus.com is the customer oriented information.

For the styleguide we need backend oriented information.

It  should provide information so that other can chose what support model to implement and what support offers to implement.

------------

Maybe we should start with creating a page with describe the purpose of the styleguide.

I will try to do it.

But this site should be the "company survival kit". In the case of complete memory lost, we should be able to use the guide and rebuild the project :)

Reply:
> But this site should be the "company survival kit". In the case of complete memory lost, we should be able to use the guide and rebuild the project :)

That makes sense.

I can copy the info from the website to the Styleguide, but still, if there are any changes to the website we still have to make updates to the Styleguide.

And the thing here is that it just adds additional work..